### PR TITLE
Rollup tutorial

### PR DESCRIPTION
**Pull request by sbvegan**
https://github.com/ethereum-optimism/docs/issues/627

### CODE REVIEW

File: pages/builders/chain-operators/tutorials/create-l2-rollup.mdx [hunk 1]
```diff
@@ -126,19 +126,18 @@ You can also use other EVM-compatible blockchains, but you may run into unexpect
 If you want to use an alternative network, make sure to carefully review each command and replace any Sepolia-specific values with the values for your network.
 
 Since you're deploying your OP Stack chain to Sepolia, you'll need to have access to a Sepolia node.
-You can either use a node provider like [Alchemy](https://www.alchemy.com/) (easier) or run your own Sepolia node (harder).
+You can either use a node provider (easier) or run your own Sepolia node (harder).
 
-## Build the Source Code
+<Callout>
+You will need access to the Layer 1 execution and consensus layers.
```
Comment (tynes) [inline pages/builders/chain-operators/tutorials/create-l2-rollup.mdx:132]: the consensus layer requirement is for OP Node posting blobs?

Comment (sbvegan) [reply pages/builders/chain-operators/tutorials/create-l2-rollup.mdx:132]: Yes, I'm pretty sure the `l1.beacon` endpoint is a hard requirement. Although this testnet rollup will be calldata. At the end, there should be call outs for how to enable op stack features: snap sync, span batches, and how to switch to using blobs.

**Comment by sbvegan** [comment]
More context on the latest best practices: https://github.com/ethereum-optimism/optimism/pull/10678

**Comment by sbvegan** [comment]
This tutorial... i think we're in a better state with the monorepo tooling to take a much better approach here.

**Pull request closed without merging**